platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+
+:::note
+This script and instructions can also be used to visualize the structures from nf-core/proteinfold runs performed with your own public or private data. 
+:::
+
+#### Create an AWS Batch compute environment 
+
+Studios requires an AWS Batch compute environment. If you do not have an existing compute environment available, create one with the following attributes:
+
+- **Region**: To minimize costs, your compute environment should be in the same region as your data. To browse the nf-core AWS megatests public data optimally, select **eu-west-1**.
+- **Provisioning model**: Use **On-demand** EC2 instances. 
+- Studios does not support AWS Fargate. Do not enable **Use Fargate for head job**. 
+- At least 2 available CPUs and 8192 MB of RAM. 
+
+#### Add data using Data Explorer
+
+For the purposes of this guide, add the proteinfold results (H1065 sequence) from the nf-core AWS megatests S3 bucket to your workspace using Data Explorer:
+
+1. From the **Data Explorer** tab, select **Add cloud bucket**. 
+1. Specify the bucket details:
+    - **Provider**: AWS
+    - **Bucket path**: `s3://nf-core-awsmegatests/proteinfold/results-9bea0dc4ebb26358142afbcab3d7efd962d3a820`
+    - A unique **Name** for the bucket, such as `nf-core-awsmegatests-proteinfold-h1065`.
+    - **Credentials**: Select **Public** from the dropdown menu.
+    - An optional bucket **Description**.
+1. Select **Add**.
+
+:::info 
+To use your own pipeline data for interactive visualization, add the cloud bucket that contains the results of your nf-core/proteinfold pipeline run. See [Add a cloud bucket](./quickstart-demo/add-data.mdx#add-a-cloud-bucket) for more information. 
+:::
+
+#### Create a Jupyter notebook studio
+
+From the **Data Studios** tab, select **Add a data studio** and complete the following:
+- In the **Compute & Data** tab:
+    - Select your AWS Batch compute environment. 
+    - Optional: Enter CPU and memory allocations. The default values are 2 CPUs and 8192 MB memory (RAM).
+        :::note
+        Studios compete for computing resources when sharing compute environments. Ensure your shared compute environment has sufficient resources to run both your pipelines and studio sessions. 
+        :::
+    - Mount data using Data Explorer: Mount the S3 bucket or directory path that contains the nf-core AWS megatests proteinfold data, or the pipeline work directory of your nf-core/proteinfold run. 
+- In the **General config** tab:
+    - Select the latest **Jupyter** container image template from the list.
+    - Optional: Enter a unique name and description for the data studio. 
+    - Check **Install Conda packages** and paste the following Conda environment YAML snippet:
+    ```yaml 
+    channels:
+    - schrodinger
+    - conda-forge
+    - bioconda
+    dependencies:
```

Comment:
I'm a fan of hard pinning dependencies, but have been burned by current-at-time-of-publication materials becoming out of data over time. Is it worth having a callout saying "~this stuff works as of DATE, but may change over time. See <some_external_link_listing_EOL/support_timelines> for exact details."